tests/components/remote/test_device_action.py

```diff
@@ -89,7 +91,7 @@ async def test_action(hass, calls, enable_custom_integrations):
                 {
                     "trigger": {"platform": "event", "event_type": "test_event1"},
                     "action": {
-                        "domain": DOMAIN,
+                        "domain": Platform.REMOTE,
```

Comment:
Before merge, is this appropriate?

Reply:
No, I don't think so. Device automations can be created by other integrations than entity integrations so other domains than entity platform domains are applicable here. So we should just use a domain constant.